From 57575cbdc0b7816af8a499afe5b8ff08ece3a619 Mon Sep 17 00:00:00 2001
From: Sergey Fedorov <barracuda@macos-powerpc.org>
Date: Mon, 9 Jun 2025 00:43:18 +0800
Subject: [PATCH 12/12] callbacks.c: do not use gcc pragmas

For some reason gcc dislikes pragmas here. The same effect can be had
by passing the flag, so drop pragmas.
---
 plugins/gtkui/callbacks.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/plugins/gtkui/callbacks.c b/plugins/gtkui/callbacks.c
index be8052752..1bac47e06 100644
--- a/plugins/gtkui/callbacks.c
+++ b/plugins/gtkui/callbacks.c
@@ -216,10 +216,7 @@ on_mainwin_key_press_event (GtkWidget *widget, GdkEventKey *event, gpointer user
             act->callback2 (act, ctx);
             return TRUE;
         }
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
         else if (act && act->callback) {
-#pragma GCC diagnostic pop
             gtkui_exec_action_14 (act, -1);
         }
     }
-- 
2.49.0

